back/app/policies/projects_allowed_input_topic_policy.rb

```diff
@@ -12,6 +12,14 @@ def resolve
     end
   end
 
+  def index?
```

Comment:
Why did you need to add it?

Reply:
To add permission for active admin or moderator.

Reply:
I mean the ticket is just about renaming. Why did you change permissions? Was it a bug? An improvement?

Reply:
In discussion with Sebastien, we realised there is no reason a not-signed in user should be able to call any action in the `projects_allowed_inputs_topics` controller.

Reply:
An improvement. Sorry, i should have made that clear. It comes from discussion with @sebastienhoorens, following his review.

Reply:
I see, it makes sense. Just one tiny improvement. As we add more and more methods with the same code here, it makes more and more sense to introduce a private method e.g.
```ruby
private

def active_admin_or_moderator?
  user&.active? && user&.active_admin_or_moderator?(record.project_id)
end
```
and call this method in public methods.

Reply:
This is done. Thanks!